test/common.sh

```diff
@@ -405,11 +424,28 @@ function create_delete_tests() {
       "t|13|37|82"
     rest_test "[rest] app create & delete (app-id=82)" \
       "/v2/applications/82?pretty" \
+      404 \
+      true \
+      ''
+    rest_test "[rest] app create & delete (app-id=82)" \
+      "/v2/applications/82?pretty&include-deleted=true" \
```

Comment:
I wasn't able to figure out how to get OpenAPI/oapi-codegen to use `include-deleted` as a flag.